mmv1/third_party/terraform/data_sources/data_source_google_compute_instance_group_manager.go

```diff
@@ -0,0 +1,50 @@
+package google
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceGoogleComputeInstanceGroupManager() *schema.Resource {
+
```

Comment:
You likely still need to add the fields `name`, `project`, `self_link`, and `zone`

Reply:
Added the fields